erroranalysis/erroranalysis/analyzer/error_analyzer.py

```diff
@@ -17,10 +17,12 @@
     compute_matrix as _compute_matrix
 from erroranalysis._internal.surrogate_error_tree import \
     compute_error_tree as _compute_error_tree
+from erroranalysis._internal.utils import generate_random_unique_indexes
 from erroranalysis._internal.version_checker import check_pandas_version
 from erroranalysis.report import ErrorReport
 
 BIN_THRESHOLD = MatrixParams.BIN_THRESHOLD
+IMPORTANCES_THRESHOLD = 50000
```

Comment:
In the scikit-learn github issue, they mentioned try with samples of 500k. 
Will 50k be too small? Comparing to 2.5m rows of data

Reply:
the test "test_large_data_importances" took
146 seconds on 500k datapoints
6 seconds on 50k datapoints
I think this is a sufficient gap in time to use 50k over 500k, especially since this is just used as a guideline for choosing features.